<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QMainWindow" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reader</string>
    </property>
   </widget>
   <widget class="QComboBox" name="input_mode">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>EventGenerator</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AraSim</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_file">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_file">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>831</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>700</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="config_file_name">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>700</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>config.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>700</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Config File Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Configuration</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>581</width>
      <height>461</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>579</width>
       <height>459</height>
      </rect>
     </property>
     <widget class="QComboBox" name="vertex1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Minuit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ASM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Interferometry</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCMC</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Coordinate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Pol</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VertexFinder</string>
      </property>
     </widget>
     <widget class="QComboBox" name="time1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Threshold</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Simple</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaus</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CSW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cross correlation</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="pol1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>78</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Vpol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hpol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Both</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="coordinate1">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Cartesian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spherical</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TimeFinder</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pol</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TimeFinder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VertexFinder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Coordinate</string>
      </property>
     </widget>
     <widget class="QComboBox" name="pol2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>78</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Vpol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hpol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Both</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="time2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Threshold</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Simple</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaus</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CSW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cross correlation</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="vertex2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Minuit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ASM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Interferometry</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCMC</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="coordinate2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>210</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Cartesian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spherical</string>
       </property>
      </item>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>561</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pol</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TimeFinder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VertexFinder</string>
      </property>
     </widget>
     <widget class="QComboBox" name="pol3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>78</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Vpol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hpol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Both</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="time3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Threshold</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Simple</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gaus</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CSW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cross correlation</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="vertex3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Minuit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ASM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Interferometry</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCMC</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>380</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Coordinate</string>
      </property>
     </widget>
     <widget class="QComboBox" name="coordinate3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>370</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Cartesian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spherical</string>
       </property>
      </item>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>execute</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>420</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>execute</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
